incident/utils/category_field_values.py

```diff
@@ -0,0 +1,283 @@
+            field,
+            item.pk
+        )
+        value = item.title.capitalize()
+        val_list.append(
+            '<li class="details-table__list-item"><a href="{}" class="text-link">{}</a></li>'.format(
+            link,
+            value
+        ))
+
+    return '<ul class="details-table__list">{}</ul>'.format('\n'.join(val_list))
+
+def equipments_list_html_val(page, field):
+    # If no value for the attribute, return blank
+    if not len(getattr(page, field).all()):
+        return ''
+
+    val_list = []
+    for item in getattr(page, field).all():
+        link = '{}?{}={}'.format(
+            page.get_parent().get_url(),
+            field,
+            item.equipment.pk
+        )
+        value = '{}, {}'.format(item.equipment.name, item.quantity)
+        val_list.append(
+            '<li class="details-table__list-item"><a href="{}" class="text-link">{}</a></li>'.format(
+            link,
+            value
+        ))
+
+    return '<ul class="details-table__list">{}</ul>'.format('\n'.join(val_list))
+
+def date_html_val(page, field):
+    # If no value for the attribute, return blank
+    if not getattr(page, field):
+        return ''
+
+    link = '{url}?{field}_lower={val}&{field}_upper={val}'.format(
+        url=page.get_parent().get_url(),
+        field=field,
+        val=getattr(page, field).strftime('%Y-%m-%d')
+    )
+    value = getattr(page, field).strftime('%B %-d, %Y')
+
+    return '<a href="{}" class="text-link"><time datetime="{}">{}</time></a>'.format(
+        link,
+        getattr(page, field).isoformat(),
+        value
+    )
+
+
+# Handle all the different category fields html values individually
+# since a lot of them small small differences that can't all be
+# covered using the generic functions above
+
+def arrest_status_html_val(page, field):
+	return basic_html_val(page, field)
+
+def status_of_charges_html_val(page, field):
+	return basic_html_val(page, field)
+
+def arresting_authority_html_val(page, field):
+	# If no value for the attribute, return blank
+    if not getattr(page, field):
+        return ''
+
+    link = '{}?{}={}'.format(
+        page.get_parent().get_url(),
+        field,
+        getattr(page, field).pk
+    )
+    value = getattr(page, field).title.capitalize()
+
+    return '<a href="{}" class="text-link">{}</a>'.format(
+        link,
+        value
+    )
+
+def current_charges_html_val(page, field):
+	return list_html_val(page, field)
+
+def dropped_charges_html_val(page, field):
+	return list_html_val(page, field)
+
+def detention_date_html_val(page, field):
+	return date_html_val(page, field)
+
+def release_date_html_val(page, field):
+	return date_html_val(page, field)
+
+def unnecessary_use_of_force_html_val(page, field):
+	# If no value for the attribute and not main category, return blank
+    main_category = page.get_main_category()
+    if not getattr(page, field) and main_category.slug != 'arrest-detention':
```

Comment:
So what I was trying to do here is [this](https://github.com/freedomofpress/pressfreedomtracker.us/blob/develop/incident/templates/incident/_arrest_rows.html#L105). It did make me squirm a bit too, but I wanted to preserve the current behaviour so I added it. The current behaviour seems to be if either it is main category or the value exists, then show the field in the table. So I just copied exactly the logic in the template because I think there were only 2-3 such cases.